diagnostic.md

```diff
@@ -14,6 +19,9 @@ list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
 <!-- your answer starts here -->
+select $('OL') using jQuery
+give the OL a class and use jQuery to call that class if there were lots of lists
+and you wanted to select this particular list
```

Comment:
ok, but how would you actually get `li` from `OL`? One way would be `$('ol > li');`, another is `$('ol').children();`